external/java/src/main/java/ch/epfl/dedis/calypso/CALYPSO.md

```diff
@@ -0,0 +1,46 @@
+# Calypso
+
+Some notes on how to use Calypso in ByzCoin. Calypso allows you to do the following:
+
+Store encrypted data together with a decryption key on ByzCoin, in a way that the decryption key is only accessible
+by trusted parties and only after they authenticated to ByzCoin and proved that they have access.
+
+Given a document with a text `secret` that needs to be put on ByzCoin, the following steps are needed:
+
+1. create a symmetric key and encrypt the `secret` using this symmetric key
+2. encrypt the symmetric using the Long Term Secret public key
+3. send the encrypted `secret` and the encrypted symmetric key to ByzCoin, together with a list of access control rules 
```

Comment:
we send the document and the encrypted key to ByzCoin? I thought it's only the encrypted key